be/src/runtime/stream_load/stream_load_executor.cpp

```diff
@@ -111,6 +112,15 @@ Status StreamLoadExecutor::execute_plan_fragment(StreamLoadContext* ctx) {
                 ctx->write_data_cost_nanos = MonotonicNanos() - ctx->start_write_data_nanos;
                 ctx->promise.set_value(status);
 
+                std::vector<TabletSharedPtr> tablets;
```

Comment:
why persist here?

Reply:
Have moved it to where sender is closed.